File: js/programmers/level_0/문자열 안에 문자열.js
// Lv. 0 문자열 안에 문자열

/**
 * @param {string} str1
 * @param {string} str2
 * @returns {number}
 */

const solution = (str1, str2) => {
    return str1.includes(str2) ? 1 : 2;
};

File: java/programmers/level_0/src/문자열안에_문자열.java
public class 문자열안에_문자열 {
//문자열 str1, str2가 매개변수로 주어집니다. str1 안에 str2가 있다면 1을 없다면 2를 return하도록 solution 함수를 완성해주세요.

  public static int solution(String str1, String str2) {
    int answer = 0;

    for (int i = 0; i < str1.length(); i++) {
      if (str1.contains(str2)) {
        answer = 1;
      } else {
        answer = 2;
      }
    }
    return answer;
  }

  public static void main(String[] args) {
    System.out.println(solution("helloajowejhvajcookiewie", "cookie"));
  }
}
